lib/percy/stats.rb

```diff
@@ -1,5 +1,6 @@
 require 'datadog/statsd'
 require 'time'
+require 'pry'
```

Comment:
Drop? This is only a dev dependency so it might actually break things to require this.